### PR TITLE
fix: change postgis function to returns table

### PR DESCRIPTION
**Pull request by steve-chavez**
Mitigates the error reported on https://github.com/supabase/postgrest-js/issues/453

### CODE REVIEW

File: apps/docs/pages/guides/database/extensions/postgis.mdx [hunk 1]
```diff
@@ -230,7 +230,7 @@ When you are working on a map-based application where the user scrolls through y
 
 ```sql
 create or replace function restaurants_in_view(min_lat float, min_long float, max_lat float, max_long float)
-returns setof record
+returns table (id public.restaurants%TYPE, name public.restaurants%TYPE, location text, dist_meters float)
```
Comment (dshukertjr) [inline apps/docs/pages/guides/database/extensions/postgis.mdx:233]: Would we need to add the column name as well before the `%` sign? Also, this particular function does not return distance, so we can delete that one. 

```suggestion
returns table (id public.restaurants.id%TYPE, name public.restaurants.name%TYPE, location text)
```

**Review by dshukertjr — COMMENTED** [review]
There is another sample postgres function that needs similar fixing. Let me fix it and push it to this branch right now.

File: apps/www/_blog/2023-03-01-geo-queries-with-postgis-in-ionic-angular.mdx [hunk 2]
```diff
@@ -127,21 +127,21 @@ Those are some powerful calculations, and we can easily use them through the Pos
 ```sql
 -- create database function to find nearby stores
 create or replace function nearby_stores(lat float, long float)
-returns setof record
+returns table (id public.stores.id%TYPE, name public.stores.name%TYPE, description public.stores.description%TYPE, lat float, long float, dist_meters float)
```
Comment (dshukertjr) [inline apps/www/_blog/2023-03-01-geo-queries-with-postgis-in-ionic-angular.mdx:130]: The update in this article is the same as the PostGis guide, and changes the return type from `setof record` to `table`.

File: apps/www/_blog/2023-03-01-geo-queries-with-postgis-in-ionic-angular.mdx [hunk 3]
```diff
@@ -127,21 +127,21 @@ Those are some powerful calculations, and we can easily use them through the Pos
...
 language sql
 as $$
-  select id, name, description, st_astext(location) as location, st_distance(location, st_point(long, lat)::geography) as dist_meters
+  select id, name, description, st_y(location::geometry) as lat, st_x(location::geometry) as long, st_distance(location, st_point(long, lat)::geography) as dist_meters
```
Comment (dshukertjr) [inline apps/www/_blog/2023-03-01-geo-queries-with-postgis-in-ionic-angular.mdx:133]: The change here is to match the model defined here
https://github.com/supabase/supabase/blob/master/apps/www/_blog/2023-03-01-geo-queries-with-postgis-in-ionic-angular.mdx#L250

**Review by dshukertjr — COMMENTED** [review]
There was an blog article that uses the same code, so included the update of the article in this PR as well.

**Comment by steve-chavez** [comment]
@dshukertjr Sorry for the late reply! Great feedback!

This looks ready to merge now.

**Review by dshukertjr — APPROVED** [review]

**Pull request merged**